src/airflow_docker/ext/__init__.py

```diff
@@ -0,0 +1,49 @@
+import functools
+import importlib
+
+import airflow
+from airflow import configuration
+
+from airflow_docker.utils import get_config
+
+
+def delegate_to_extensions(self, method_name, *args, **kwargs):
+    config = get_config()["airflow-docker"]
+    for extension in self._extensions:
+        method = getattr(extension, method_name, None)
+        if not method:
+            continue
+
+        method(self, config, *args, **kwargs)
+
+
+def by_name(item):
+    return item.__name__
+
+
+@functools.lru_cache()
+def load_extensions():
+    extension_paths = ""
+    if configuration.has_option("extensions", "paths"):
```

Comment:
Can you replace this configuration access with a function in [airflow_docker.conf](https://github.com/huntcsg/airflow-docker/blob/master/src/airflow_docker/conf.py)?

And also, should the option be under the top level `airflowdocker` key instead of `extensions`?